Example/IceCream_Example.xcodeproj/project.pbxproj

```diff
@@ -358,11 +358,13 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = IceCream_Example/IceCream_Example.entitlements;
 				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = EQRQCAG846;
```

Comment:
might want to remove this and the bundleID from any PR that you send to the main repo, but for our own use case it's nice that it made It in.

Reply:
Yeah, I think if it doesn't seem very maintained and you want to safe time, merging back into our own fork is totally fine. you could still send a PR over to them just to see if you get any movement, but no need to spend a lot of time on it.